src/wp-admin/includes/translation-install.php

```diff
@@ -35,23 +34,26 @@ function translations_api( $type, $args = null ) {
 	$res = apply_filters( 'translations_api', false, $type, $args );
 
 	if ( false === $res ) {
-		$url = $http_url = 'https://api.wordpress.org/translations/' . $type . '/1.0/';
-
+		$stats = array(
+			'wp_version' => '4.9.9', //since CP is based on 4.9.9
```

Comment:
This needs to be `$wp_version` instead.

We also don't need to send the `wp_version` to the API server, so you can remove it here...

Reply:
so we are still defining that variable?

Reply:
yes